.travis.yml

```diff
@@ -5,6 +5,11 @@ python:
   - "3.5"
 addons:
   postgresql: "9.4"
+  apt:
+    sources:
+      - travis-ci/sqlite3
+    packages:
+      - sqlite3
 
 services:
   - postgresql
```

Comment:
Travis uses the older version of sqlite3. This enables the new installation of a more current version than the default version